Classes/Domain/NodeCreation/NodeCreationService.php

```diff
@@ -59,8 +59,9 @@ public function apply(RootTemplate $template, NodeInterface $node, CaughtExcepti
 
     private function applyTemplateRecursively(Templates $templates, NodeInterface $parentNode, CaughtExceptions $caughtExceptions): void
     {
+        $parentNodesAutoCreatedChildNodes = $parentNode->getNodeType()->getAutoCreatedChildNodes();
         foreach ($templates as $template) {
-            if ($template->getName() && $parentNode->getNodeType()->hasAutoCreatedChildNode($template->getName())) {
+            if ($template->getName() && isset($parentNodesAutoCreatedChildNodes[$template->getName()->__toString()])) {
```

Comment:
`hasAutoCreatedChildNode` actually has a bug: It looks up the NodeName parameter against the raw configuration instead of the transliterated NodeName